open_schools_platform/user_management/users/tests/views/test_users_exceptions.py

```diff
@@ -0,0 +1,179 @@
+import datetime
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from open_schools_platform.user_management.users.services import create_user, create_token
+
+
+class UserExceptionsTests(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+        self.token_creation_url = "/api/user-management/users/token"
+        self.token_verification_url = "/api/user-management/users/token/{}/verify"
+        self.user_creation_url = "/api/user-management/users/"
+        self.token_data_resend_url = "/api/user-management/users/token/{}"
+        self.sms_resend_url = "/api/user-management/users/token/{}/resend"
+
+    def test_user_already_created(self):
+        credentials = {
+            "phone": "+79020000000",
+            "password": "qwe",
+            "name": "Vasya",
+        }
+
+        create_user(**credentials)
+
+        data = {
+            "phone": "+79020000000",
+            "recaptcha": "123456"
+        }
+        response = self.client.post(self.token_creation_url, data)
+        self.assertEqual(409, response.status_code)
+
+    def test_firebase_response_is_not_200(self):
+        data = {
+            "phone": "+79020000000",
+            "recaptcha": "123456"
+        }
+        response = self.client.post(self.token_creation_url, data)
+        self.assertEqual(400, response.status_code)
+
+    def test_token_with_such_id_does_not_exist(self):
```

Comment:
You can merge some tests.
For example, test_token_with_such_id_does_not_exist can test all urls with where token can not exist